common/src/main/java/com/wynntils/mc/mixin/BossHealthOverlayMixin.java

```diff
@@ -31,46 +32,52 @@
                             target =
                                     "Lnet/minecraft/network/protocol/game/ClientboundBossEventPacket;dispatch(Lnet/minecraft/network/protocol/game/ClientboundBossEventPacket$Handler;)V"))
     private void updatePre(ClientboundBossEventPacket packet, ClientboundBossEventPacket.Handler handler) {
+        Map<UUID, LerpingBossEvent> _events = events;
```

Comment:
If it turns out that we need to keep this, can you please rename it? I'm old enough to remember the school of Javaa naming that required instance variables to be prefixed with `_`, so I instinctively think that `_events` is a field, and `events` is a local variable. 

Maybe something like `eventMap`, or `bossEvents`, or even the cheese `myEvents`...